app_backboneJS/song/song-list-view.js

```diff
@@ -20,5 +29,11 @@ const SongsListView = Backbone.View.extend({
     });
     return this;
   },
+  closeListOfSongs() {
+    const elem = window.querySelector('#playlist-songs');
```

Comment:
nu căuta global în DOM după elemente, foloseşte this.el şi caută doar elementele generate de template-ul view-ului